src/features/deposit/components/DepositUIMachineProvider.tsx

```diff
@@ -152,7 +153,7 @@ export function DepositUIMachineProvider({
                 assert(accountId != null, "Account ID is not defined")
                 assert(chain != null, "Chain is not defined")
                 const address = await depositNearService.generateDepositAddress(
-                  accountId,
+                  userAddressToDefuseUserId(accountId),
```

Comment:
`depositNearService.generateDepositAddress` should have `DefuseUserId` requirement, so no one could accidentally pass incorrect string.